packages/telemetry/src/index.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+/* globals globalThis, process */
+import { MeterProvider } from '@opentelemetry/metrics';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+
+/**
+ * @typedef {Object} Powers
+ * @property {{ log: Console['log'] }} console
+ * @property {NodeJS.ProcessEnv} env
+ */
+
+/**
+ * @param {Partial<Powers>} param0
+ */
+const getPrometheusMeterProvider = ({
+  console = globalThis.console,
+  env = process.env,
+} = {}) => {
+  const { OTEL_EXPORTER_PROMETHEUS_PORT } = env;
+  if (!OTEL_EXPORTER_PROMETHEUS_PORT) {
+    // No Prometheus config, so don't install.
+    return undefined;
+  }
+
+  const port =
+    parseInt(OTEL_EXPORTER_PROMETHEUS_PORT || '', 10) ||
+    PrometheusExporter.DEFAULT_OPTIONS.port;
+
+  const exporter = new PrometheusExporter(
+    {
+      port,
+    },
+    () => {
+      console.log(
```

Comment:
Might be better shunted to stderr in case stdout has data that can be piped.

Reply:
Hoist on my own petard.

Reply:
I thought they were called “flag standards”. What a language.